### PR TITLE
Ruby 2.7

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
Just checking out how Molinillo works with ruby 2.7 in CI.

### CODE REVIEW

File: spec/resolver_spec.rb [hunk 1]
```diff
@@ -182,7 +182,7 @@ def self.save!(path, name, index, requirements, resolved)
 Unable to satisfy the following requirements:
 
 - `json (>= 1.7.7)` required by `berkshelf (2.0.7)`
-- `json (>= 1.4.4, <= 1.7.7)` required by `chef (10.26)`
+- `json (<= 1.7.7, >= 1.4.4)` required by `chef (10.26)`
```
Comment (segiddins) [inline spec/resolver_spec.rb:185]: why is this changing?

Comment (deivid-rodriguez) [reply spec/resolver_spec.rb:185]: I'll reword the commit messages with proper links to make it clear!

Comment (deivid-rodriguez) [reply spec/resolver_spec.rb:185]: @segiddins I added a proper explanation in https://github.com/CocoaPods/Molinillo/pull/139/commits/d42b00a38a140dd95ce2d739691762c026dd156e, and also added a few other changes to get CI green.

**Comment by deivid-rodriguez** [comment]
Thanks!

**Pull request merged**